pkg/extensions/extensions.go

```diff
@@ -52,16 +54,72 @@ func EnableExtensions(extension *ExtensionConfig, log log.Logger, rootDir string
 	} else {
 		log.Info().Msg("CVE config not provided, skipping CVE update")
 	}
+
+	if extension.Sync != nil {
+		defaultPollInterval, _ := time.ParseDuration("1h")
+		for _, registryCfg := range *extension.Sync {
+			if registryCfg.PollInterval < defaultPollInterval {
+				registryCfg.PollInterval = defaultPollInterval
+
+				log.Warn().Msg("Sync registries interval set to too-short interval <= 1h, changing update duration to 1 hour and continuing.") // nolint: lll
+			}
+		}
+
+		var serverCert string
+
+		var serverKey string
+
+		var CACert string
+
+		if config.HTTP.TLS != nil {
```

Comment:
What are you trying to achieve here? I see that you are copying these certs. But anything that is needed by the extension should be in the ExtensionConfig itself.

Reply:
I thought that repeating information in config was bad, but if you agree to repeat this information in the ExtensionConfig, then I don't need the config refactoring above.